docs/_documentations/mdt-vsc-installinfo.md

```diff
@@ -24,12 +24,10 @@ The following images are pulled. These images together form the Codewind back en
 - `eclipse/codewind-pfe-amd64`
 
 ## Using Codewind
-When the installation is complete, the extension is ready to use, and you are prompted to open the Codewind workspace.
-
-You can open the `codewind-workspace` or a project within the workspace as your VS Code workspace. For more information, see [Getting started: Codewind for VS Code](mdt-vsc-getting-started.html).
+When the installation is complete, the extension is ready to use. 
 
 ## Working with workspaces
```

Comment:
We should probably remove this 'Working with workspaces' heading, since the text under it is no longer talking about workspaces, and is short. I think it's fine to leave the text as a second paragraph under the 'Using Codewind' heading